ui/packages/lib/src/hooks/useApi.js

```diff
@@ -112,7 +115,12 @@ export const useApi = (
           )
       )
         .then(result => {
-          if (!didCancel) dispatch({ type: "FETCH_SUCCESS", payload: result });
+          if (!didCancel)
+            dispatch({
+              type: "FETCH_SUCCESS",
+              payload: result.body,
```

Comment:
I think changing `result` to `result.body` as the value for payload will break the API. How about keeping the `payload` that still use `result`, and then introduce new field `body` and `headers` which use `result.body` and `result.headers`?

Btw, you also need to update the version of @mlp-ui library in package.json

Reply:
can you elaborate in what case it will break the API?

Reply:
My bad, I failed to notice that you've also changed fetchJson in which you return result as body's value. It won't break the API. Data will still use the same `result`.